### DISCUSSION TITLE
Recruiting for GitHub's Accessibility Discussion Panel!

### DISCUSSION THREAD

**Discussion by realkimpossible** [community/community#56353] [Accessibility]
### Select Topic Area

General

### Body

Do you use technologies like screen readers, speech to text, dark mode, eye tracking, etc. for your daily work? GitHub wants to hear from you!

GitHub will be hosting a panel of GitHub customers who have lived experiences using assistive technologies. You’ll be compensated for your feedback. If you’d like to participate, [please sign up here](https://airtable.com/shrNwQ4X247Bm4drZ) to let us know you’re interested!

Sincerely,
GitHub Research

**Comment by Zalathar** [comment]
It may be worth double-checking that email link. On iOS my mail client seems to think it’s invalid for some reason.

**Reply by realkimpossible** [reply]
Fixed, thank you!

**Comment by manoharkharvi43** [comment]
GitHub is looking for customers who use assistive technologies like screen readers, speech-to-text, dark mode, eye tracking, etc., for their daily work. They want to hear about your experiences and will compensate you for your feedback.

By participating in this panel, you can help GitHub improve their platform's accessibility and cater better to users with diverse needs. Your input is valuable, and GitHub recognizes the importance of inclusivity in technology.

**Comment by Binary-Ninja-007** [comment]
Good.
